[Waiting for rollout to production] Adding new APNS credential properties(BundleId and TeamId)

amolr opened the pull request:
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

Commit:
Adding new APNS credential properties

fearthecowboy commented:
@amolr 

What exactly are you doing here?

Are you intending to add a new API version?

You're going to have to update the readme.md file and make sure the CI steps pass.

Commit:
Removed extra comma

Commit:
Updating the readme file

amolr commented:
@fearthecowboy  I have made the required changes. Can you please review it

fearthecowboy commented:
@amolr  - When updating to a new API version, we need to clean up the issues in the swagger file, and it's also a good opportunity to conform to the latest standards for the ARM rest specs.

(FYI, when you  create the PR, there's a checklist you're supposed follow -- see the first post!)

We've introduced a file that contains the common base types (Resource, TrackedResource,  etc) which you should reference from your swagger file for common definitions : https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v1/types.json

We're also going to have to get an ARM review from @ravbhatnagar 

You should run autorest and look at the errors/warnings that are generated:

``` powershell
cd C:\tmp\amolr\azure-rest-api-specs\specification\notificationhubs\resource-manager
autorest --azure-validator readme.md
``` 
which produces:

``` haskell

WARNING (PostOperationIdContainsUrlVerb/R2066/SDKViolation): OperationId should contain the verb: 'checknamespaceavailability' in:'Namespaces_CheckAvailability'. Consider updating the operationId
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:42:8 ($.paths["/subscriptions/{subscriptionId}/providers/Microsoft.NotificationHubs/checkNamespaceAvailability"].post.operationId)

WARNING (PostOperationIdContainsUrlVerb/R2066/SDKViolation): OperationId should contain the verb: 'checknameavailability' in:'Name_CheckAvailability'. Consider updating the operationId
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:76:10 ($.paths["/subscriptions/{subscriptionId}/providers/Microsoft.NotificationHubs/checkNameAvailability"].post.operationId)

WARNING (PostOperationIdContainsUrlVerb/R2066/SDKViolation): OperationId should contain the verb: 'checknotificationhubavailability' in:'NotificationHubs_CheckAvailability'. Consider updating the operationId
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:643:10 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NotificationHubs/namespaces/{namespaceName}/checkNotificationHubAvailability"].post.operationId)

WARNING (PostOperationIdContainsUrlVerb/R2066/SDKViolation): OperationId should contain the verb: 'checkhubavailability' in:'Hubs_CheckAvailability'. Consider updating the operationId
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:691:10 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NotificationHubs/namespaces/{namespaceName}/checkHubAvailability"].post.operationId)

WARNING (PutRequestResponseScheme/R2017/SDKViolation): A PUT operation request body schema should be the same as its 200 response schema, to allow reusing the same entity between GET and PUT. If the schema of the PUT request body is a superset of the GET response body, make sure you have a PATCH operation to make the resource updatable. Operation: 'Namespaces_CreateOrUpdate' Request Model: 'NamespaceCreateOrUpdateParameters' Response Model: 'NamespaceResource'
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:106:8 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NotificationHubs/namespaces/{namespaceName}"].put)

WARNING (PutRequestResponseScheme/R2017/SDKViolation): A PUT operation request body schema should be the same as its 200 response schema, to allow reusing the same entity between GET and PUT. If the schema of the PUT request body is a superset of the GET response body, make sure you have a PATCH operation to make the resource updatable. Operation: 'Namespaces_CreateOrUpdateAuthorizationRule' Request Model: 'SharedAccessAuthorizationRuleCreateOrUpdateParameters' Response Model: 'SharedAccessAuthorizationRuleResource'
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:290:8 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NotificationHubs/namespaces/{namespaceName}/AuthorizationRules/{authorizationRuleName}"].put)

WARNING (PutRequestResponseScheme/R2017/SDKViolation): A PUT operation request body schema should be the same as its 200 response schema, to allow reusing the same entity between GET and PUT. If the schema of the PUT request body is a superset of the GET response body, make sure you have a PATCH operation to make the resource updatable. Operation: 'NotificationHubs_CreateOrUpdate' Request Model: 'NotificationHubCreateOrUpdateParameters' Response Model: 'NotificationHubResource'
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:735:8 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NotificationHubs/namespaces/{namespaceName}/notificationHubs/{notificationHubName}"].put)

WARNING (PutRequestResponseScheme/R2017/SDKViolation): A PUT operation request body schema should be the same as its 200 response schema, to allow reusing the same entity between GET and PUT. If the schema of the PUT request body is a superset of the GET response body, make sure you have a PATCH operation to make the resource updatable. Operation: 'NotificationHubs_CreateOrUpdateAuthorizationRule' Request Model: 'SharedAccessAuthorizationRuleCreateOrUpdateParameters' Response Model: 'SharedAccessAuthorizationRuleResource'
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:881:8 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NotificationHubs/namespaces/{namespaceName}/notificationHubs/{notificationHubName}/AuthorizationRules/{authorizationRuleName}"].put)

WARNING (TrackedResourceListByImmediateParent/R3010/RPCViolation): The child tracked resource, 'AuthorizationRules' with immediate parent 'NamespaceResource', must have a list by immediate parent operation.
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1297:4 ($.definitions)

WARNING (TrackedResourceListByImmediateParent/R3010/RPCViolation): The child tracked resource, 'notificationHubs' with immediate parent 'NamespaceResource', must have a list by immediate parent operation.
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1297:4 ($.definitions)

WARNING (TrackedResourceListByImmediateParent/R3010/RPCViolation): The child tracked resource, 'AuthorizationRules' with immediate parent 'NotificationHubResource', must have a list by immediate parent operation.
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1297:4 ($.definitions)

WARNING (PatchInOperationName/R1007/SDKViolation): 'PATCH' operation 'Namespaces_Patch' should use method name 'Update'. Note: If you have already shipped an SDK on top of this spec, fixing this warning may introduce a breaking change.
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:165:10 ($.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.NotificationHubs/namespaces/{namespaceName}"].patch.operationId)

WARNING (TrackedResourceListByResourceGroup/R3027/RPCViolation): The tracked resource, 'NamespaceResource', must have a list by resource group operation.(This rule does not apply for tenant level resources.)
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1488:6 ($.definitions.NamespaceResource)

WARNING (TrackedResourceListByResourceGroup/R3027/RPCViolation): The tracked resource, 'SharedAccessAuthorizationRuleResource', must have a list by resource group operation.(This rule does not apply for tenant level resources.)
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1541:6 ($.definitions.SharedAccessAuthorizationRuleResource)

WARNING (TrackedResourceListByResourceGroup/R3027/RPCViolation): The tracked resource, 'NotificationHubResource', must have a list by resource group operation.(This rule does not apply for tenant level resources.)
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1863:6 ($.definitions.NotificationHubResource)

WARNING (TrackedResourceListByResourceGroup/R3027/RPCViolation): The tracked resource, 'NamespaceCreateOrUpdateParameters', must have a list by resource group operation.(This rule does not apply for tenant level resources.)
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1454:6 ($.definitions.NamespaceCreateOrUpdateParameters)

WARNING (TrackedResourceListByResourceGroup/R3027/RPCViolation): The tracked resource, 'SharedAccessAuthorizationRuleCreateOrUpdateParameters', must have a list by resource group operation.(This rule does not apply for tenant level resources.)
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1524:6 ($.definitions.SharedAccessAuthorizationRuleCreateOrUpdateParameters)

WARNING (TrackedResourceListByResourceGroup/R3027/RPCViolation): The tracked resource, 'NotificationHubCreateOrUpdateParameters', must have a list by resource group operation.(This rule does not apply for tenant level resources.)
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1844:6 ($.definitions.NotificationHubCreateOrUpdateParameters)

WARNING (TrackedResourceListBySubscription/R3028/RPCViolation): The tracked resource, 'SharedAccessAuthorizationRuleResource', must have a list by subscriptions operation.
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1541:6 ($.definitions.SharedAccessAuthorizationRuleResource)

WARNING (TrackedResourceListBySubscription/R3028/RPCViolation): The tracked resource, 'NamespaceCreateOrUpdateParameters', must have a list by subscriptions operation.
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1454:6 ($.definitions.NamespaceCreateOrUpdateParameters)

WARNING (TrackedResourceListBySubscription/R3028/RPCViolation): The tracked resource, 'SharedAccessAuthorizationRuleCreateOrUpdateParameters', must have a list by subscriptions operation.
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1524:6 ($.definitions.SharedAccessAuthorizationRuleCreateOrUpdateParameters)

WARNING (TrackedResourceListBySubscription/R3028/RPCViolation): The tracked resource, 'NotificationHubCreateOrUpdateParameters', must have a list by subscriptions operation.
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1844:6 ($.definitions.NotificationHubCreateOrUpdateParameters)

WARNING (DescriptionAndTitleMissing/R4000/SDKViolation): 'CheckNameAvailabilityResponse' model/property lacks 'description' and 'title' property. Consider adding a 'description'/'title' element. Accurate description/title is essential for maintaining reference documentation.
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1315:6 ($.definitions.CheckNameAvailabilityResponse)

WARNING (DescriptionAndTitleMissing/R4000/SDKViolation): 'Resource' model/property lacks 'description' and 'title' property. Consider adding a 'description'/'title' element. Accurate description/title is essential for maintaining reference documentation.
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1938:6 ($.definitions.Resource)

WARNING (DescriptionAndTitleMissing/R4000/SDKViolation): 'SubResource' model/property lacks 'description' and 'title' property. Consider adding a 'description'/'title' element. Accurate description/title is essential for maintaining reference documentation.
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1976:6 ($.definitions.SubResource)

WARNING (LocationMustHaveXmsMutability/S4002/SDKViolation): Property 'location' must have '"x-ms-mutability":["read", "create"]' extension defined. Resource Model: 'Resource'
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1297:4 ($.definitions)

WARNING (LocationMustHaveXmsMutability/S4002/SDKViolation): Property 'location' must have '"x-ms-mutability":["read", "create"]' extension defined. Resource Model: 'Resource'
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1297:4 ($.definitions)

WARNING (LocationMustHaveXmsMutability/S4002/SDKViolation): Property 'location' must have '"x-ms-mutability":["read", "create"]' extension defined. Resource Model: 'Resource'
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1297:4 ($.definitions)

WARNING (LocationMustHaveXmsMutability/S4002/SDKViolation): Property 'location' must have '"x-ms-mutability":["read", "create"]' extension defined. Resource Model: 'Resource'
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1297:4 ($.definitions)

WARNING (LocationMustHaveXmsMutability/S4002/SDKViolation): Property 'location' must have '"x-ms-mutability":["read", "create"]' extension defined. Resource Model: 'Resource'
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1297:4 ($.definitions)

WARNING (LocationMustHaveXmsMutability/S4002/SDKViolation): Property 'location' must have '"x-ms-mutability":["read", "create"]' extension defined. Resource Model: 'Resource'
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1297:4 ($.definitions)

WARNING (EnumInsteadOfBoolean/R3018/RPCViolation): Booleans are not descriptive and make them hard to use. Consider using string enums with allowed set of values defined. Property: NameAvailable
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1317:10 ($.definitions.CheckNameAvailabilityResponse.properties.NameAvailable)

WARNING (EnumInsteadOfBoolean/R3018/RPCViolation): Booleans are not descriptive and make them hard to use. Consider using string enums with allowed set of values defined. Property: isAvailiable
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1362:10 ($.definitions.CheckAvailabilityParameters.properties.isAvailiable)

WARNING (EnumInsteadOfBoolean/R3018/RPCViolation): Booleans are not descriptive and make them hard to use. Consider using string enums with allowed set of values defined. Property: isAvailiable
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1375:10 ($.definitions.CheckAvailabilityResult.properties.isAvailiable)

WARNING (EnumInsteadOfBoolean/R3018/RPCViolation): Booleans are not descriptive and make them hard to use. Consider using string enums with allowed set of values defined. Property: enabled
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1431:10 ($.definitions.NamespaceProperties.properties.enabled)

WARNING (EnumInsteadOfBoolean/R3018/RPCViolation): Booleans are not descriptive and make them hard to use. Consider using string enums with allowed set of values defined. Property: critical
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1435:10 ($.definitions.NamespaceProperties.properties.critical)

WARNING (AvoidNestedProperties/R2001/SDKViolation): Consider using x-ms-client-flatten to provide a better end user experience
    - file:///C:/tmp/amolr/azure-rest-api-specs/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:1526:10 ($.definitions.SharedAccessAuthorizationRuleCreateOrUpdateParameters.properties.properties)

```

Commit:
Code review and Azure validator changes

fearthecowboy commented:
I'll take a look after lunch!

Commit:
Adding common types

fearthecowboy committed:
tweaked a bit to cleanup

amolr committed:
Merge pull request #1 from fearthecowboy/amolr

tweaked a bit to cleanup

Commit:
Removing redundant operations and build errors

Commit:
Changed the name from Namespace to NamespaceResource

fearthecowboy commented:
Attn @ravbhatnagar - new API version -- need ARM signoff

This actually really very simple, in `ApnsCredentialProperties` they removed `appName` and `appId` and added `bundleId` and `teamId` properties.

All the other changes were to use the common types, and clean up the naming of operations to fit to guidelines better.

amolr commented:
@ravbhatnagar  Can you please approve this request

lmazuel commented on specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:
```diff
@@ -0,0 +1,1971 @@
+      },
+      "CheckNameAvailabilityResponse": {
+        "properties": {
+          "nameAvailable": {
```
Are you 100% this is the **exact** JSON key used? Previous API version was cased "NameAvailable".
Note that C# JSON parser is case-insensitive (and that's a problem), so even if this Swagger is working in C#, does not mean this is correct. We need to be sure this is exactly what Azure is returning.
Thanks,

amolr replied:
This is what the azure validator warning mentions....to make it camelcase. Also this is a breaking change so changing the casing shouldnt matter right?

lmazuel replied:
@amolr There is two things:
- The validator validates against **convention**. It doesn't know what the RestAPI **actually** returns.
- The Swagger **MUST** tell the truth of the RestAPI. 

If the RestAPI does not follow the convention, you will get a warning **and you can't fix it**. You might add a "warning suppression" in the Readme to avoid this message.
Changing the case in the Swagger to remove the warnings is like breaking you car dashboard to turn off the "engine check" light: this does not address the real issue.

@salameer This is the second time I need to explain the difference between validation and RestAPI accurancy, I think there is a miss of documentation to help people understand what is the right fix.

fearthecowboy replied:
@amolr -- if it's actually uppercase (ie `NameAvailable`) on the wire, we can add a suppression for error. @lmazuel  is correct, the swagger must be accurate .

lmazuel reviewed (commented):
Check changing case please.

Commit:
Changing the case

lmazuel committed:
Notification Hub Python to version 2.0.0

There is massive breaking changes that requires next Python SDK to be released as 2.0.0

lmazuel commented:
@amolr You're adding the new swagger in the "stable" folder. Could you confirm that when this PR is merged, this means that the RestAPI is GA/deployed in all regions and ready to be generated for all SDKs?
Thanks,

Commit:
Renamed property

Commit:
Merge branch 'master' of https://github.com/amolr/azure-rest-api-specs

amolr commented:
REST API is not yet deployed but we shall deploy it soon. Meanwhile we want to keep our SDKs ready so that as soon as the REST API is deployed people can start using the SDK

amolr commented:
Can you please approve this pull request in that case

fearthecowboy committed:
Update readme.md

fearthecowboy reviewed (approved):
Looks good to me. 

@amolr notify me when the API is in production, so that this can be merged.

lmazuel commented:
@AutorestCI rebuild azure-sdk-for-python

AutorestCI commented:
# Automation for azure-sdk-for-python

jhendrixMSFT commented:
@AutorestCI rebuild azure-sdk-for-go

AutorestCI commented:
# Automation for azure-sdk-for-go

salameer commented:
@amolr is the service rolled out? should this be merged now?

azuresdkci commented:
Can one of the admins verify this patch?

nschonni commented on specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:
```diff
@@ -0,0 +1,1971 @@
+          ],
+          "responses": {
+            "200": {
+              "description": "Checks if the Namespace name is a valid availabile name",
```
availabile -> available

nschonni commented on specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:
```diff
@@ -0,0 +1,1971 @@
+          ],
+          "responses": {
+            "200": {
+              "description": "Checks if the NotificationHub name is a valid availabile name",
```
availabile -> available

nschonni commented on specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:
```diff
@@ -0,0 +1,1971 @@
+          },
+          "Message": {
+            "type": "string",
+            "description": "The messsage returned when checking for namespace name availability"
```
messsage -> message

nschonni commented on specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2017-11-01/notificationhubs.json:
```diff
@@ -0,0 +1,1971 @@
+            "$ref": "#/definitions/Sku",
+            "description": "The sku of the created namespace"
+          },
+          "isAvailiable": {
```
isAvailiable -> isAvailable

nschonni reviewed (commented):
typos

salameer commented:
@amolr is this out in production already,

THis PR has been sitting for months now and will be closed unless we get an update here

salameer commented:
CLosing, please open a new one or request re opening if needed

AutorestCI commented:
# Automation for azure-sdk-for-ruby

AutorestCI commented:
# Automation for azure-sdk-for-js

AutorestCI commented:
# Automation for azure-sdk-for-java
